### PR TITLE
Math-Magician Components Features

### PR DESCRIPTION
**Pull request by DanSam5K**
## In this PR I accomplished the following:
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components and inside components, create a new Calculator.js file.
- Develop just the UI of the calculator, by using class-based components.
- Fix all linters errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (arslanbisharat) [inline src/App.js]: - [x]  The `App Component` should be a `Class Component` as specified in the project requirement. Let us refactor it into a class component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [ ] Let's not disable linter rules.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [Optional] I will advise that you might consider removing the auto-generated section from your README file and only stick to your custom edited. That changelog has nothing to do with your project. Thank you


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (DanSam5K) [reply src/App.js]: Noted

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @DanSam5K

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer. 
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by DanSam5K** [comment]
- Thank you @Emmyn5600  for pointing out my strength & @arslanbisharat for making sure I improve on my weakness. Excellent reviewers. ❤️

**Pull request merged**